test/specs/app.spec.js

```diff
@@ -3,9 +3,18 @@ import React from 'react';
 import { mount, shallow } from 'enzyme';
 import sinon from 'sinon';
 
+import UtilsHelper from "../../src/app/helper/utilsHelper";
```

Comment:
Would it be worth using a path resolver set the application base path to app then you just say from 'app/...'